src/platform/plugins/private/presentation_panel/public/panel_actions/show_config_panel_action/execute_show_config_action.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { ActionExecutionMeta } from '@kbn/ui-actions-plugin/public';
+import type { EmbeddableApiContext } from '@kbn/presentation-publishing';
+import { CONTEXT_MENU_TRIGGER } from '../triggers';
+import { ACTION_SHOW_CONFIG_PANEL } from './constants';
+import { uiActions } from '../../kibana_services';
+
+export async function executeShowConfigPanelAction(api: unknown) {
```

Comment:
Is this used anywhere? I could not find `executeShowConfigPanelAction` called anywhere in the code base. How about removing?

Reply:
🔥 Removed with [37adbd8](https://github.com/elastic/kibana/pull/208554/commits/37adbd8337340ad915294661404ce0be6bd20916)